utils.js

```diff
@@ -240,17 +265,25 @@ export async function id() {
 export async function delegate({zcap, signer, capabilityChain}) {
   capabilityChain =
     Array.isArray(capabilityChain) ? capabilityChain : [zcap.parentCapability];
-  // attach capability delegation proof
-  return sign(zcap, {
-    // TODO: map `signer.type` to signature suite
-    suite: new Ed25519Signature2018({
+  let suite;
+  if(signer.type === 'Ed25519VerificationKey2018') {
+    suite = new Ed25519Signature2018({
       signer,
       verificationMethod: signer.id
-    }),
+    });
+  } else if(signer.type === 'Ed25519VerificationKey2020') {
+    suite = new Ed25519Signature2020({
+      signer,
+      verificationMethod: signer.id
```

Comment:
```suggestion
```
(the suites no longer have a `verificationMethod` constructor param).

Reply:
Addressed xa1f58af